testsuite/src/test/java/org/ironjacamar/rars/ResourceAdapterFactory.java

```diff
@@ -345,7 +345,7 @@ public static ResourceAdapterArchive createUnifiedSecurityRar()
     * @return The resource adapter descriptor
     */
    public static ResourceAdaptersDescriptor createUnifiedSecurityDeployment(String bc, String securityDomain,
```

Comment:
Maybe do some overloaded methods, so you don't have to pass in all parameters in all tests